### PR TITLE
Adoption application reviews tests

### PR DESCRIPTION
**Pull request by Overlandcoder**
These tests are for the first bullet point of Issue #49. I _think_ the name of the test file is appropriate.

3 total tests to check that:
- verified staff can see all applications
- unverified staff cannot access the page
- all expected elements of an application are shown

Hopefully these are in line with how you'd write such tests!

### CODE REVIEW

File: test/integration/adoption_application_reviews_test.rb [hunk 1]
```diff
@@ -12,4 +12,12 @@ class AdoptionApplicationReviewsTest < ActionDispatch::IntegrationTest
       count: AdopterApplication.count, text: 'Edit Application'
     }
   end
+
+  test "unverified staff cannot access the page" do
+    sign_in users(:user_three)
+
+    get '/adopter_applications'
+
+    assert_response :redirect
```
Comment (kasugaijin) [inline test/integration/adoption_application_reviews_test.rb:21]: Could you please add an assertion to expect an alert flash after redirect? Check the application_controller.rb for the verified staff and assert that message in a flash. There are examples of this in the org_dogs_test.rb checking that unverified staff cannot access the org dogs page. I think it's handy to make sure the before action filter is working for this authorization.

File: test/integration/adoption_application_reviews_test.rb [hunk 2]
```diff
@@ -20,4 +20,16 @@ class AdoptionApplicationReviewsTest < ActionDispatch::IntegrationTest
...
     assert_response :redirect
   end
+
+  test "all expected elements of an application are shown" do
+    sign_in users(:user_two)
+
+    get '/adopter_applications'
+
+    assert_select 'a', AdopterApplication.first.dog.name
+    assert_select 'p', "Applicant: #{AdopterApplication.first.adopter_account.user.first_name}
```
Comment (kasugaijin) [inline test/integration/adoption_application_reviews_test.rb:30]: Looks good! We could clean it up a little by creating an instance var (e.g., @adopter = ...)for the AdopterApplication.first.adopter_account.user and then calling first_name and last_name on that in the assertions?

**Comment by Overlandcoder** [comment]
All the requested changes have been made. Also it turns out that the new adoption application fixture you added yesterday was causing 2 of the assertions to fail. It took me embarrassingly long to finally figure out how to fix them haha. Please let me know if something looks off!

**Review by kasugaijin — COMMENTED** [review]
Looks much better. Cleaner and less brittle! Thank you for getting to these. 
Good job merging main into your branch, too, or those would have been missed.

**Pull request merged**